src/query/storage/m3/storage.go

```diff
@@ -163,6 +163,11 @@ func (s *m3storage) Fetch(
 		fetchResult.Metadata.Resolutions = resolutions
 	}
 
+	if options.IncludeExemplars {
```

Comment:
It may be a good idea to change the `options.BlockType == models.TypeDecodedBlock` conditional inside the `FetchBlocks()`  method  on :209 to be `options.BlockType == models.TypeDecodedBlock || options.IncludeExemplars` to always force it down this path if you're expecting Exemplars?